src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -244,6 +239,18 @@ public Claim saveClaim(
         return savedClaim;
     }
 
+    private ClaimSubmissionOperationIndicators getDefaultIndicators(ClaimData claimData) {
```

Comment:
Rather than passing `ClaimData` object here, can we just pass flag `isCitizenClaim` or `isRepresentedClaim`?